app/views/hostgroups/_form.html.erb

```diff
@@ -69,7 +69,6 @@
       <p/>
       <%= render "puppetclasses/classes_parameters", { :obj => @hostgroup } %>
       <p/>
-      <%= render "common_parameters/inherited_parameters" %>
       <h6><%= _('Host group parameters') %></h6>
```

Comment:
These `<h6>` headers (and in the host form as well) are even smaller then the table headers below it, making it unclear that they are headers and not the footer of the previous table. Consider changing to H4 and/or adding underline to the header and/or borders between tables.